changeset/commit.go

```diff
@@ -26,18 +26,19 @@ import (
 )
 
 const (
-	commitIDFile      = "HEAD"
-	koDataPathEnvName = "KO_DATA_PATH"
+	headCommitIDFile   = "HEAD"
+	masterCommitIDFile = "master"
+	koDataPathEnvName  = "KO_DATA_PATH"
 )
 
 var (
 	commitIDRE = regexp.MustCompile(`^[a-f0-9]{40}$`)
 )
 
-// Get tries to fetch the first 7 digitals of GitHub commit ID from HEAD file in
+// get tries to fetch the first 7 digitals of GitHub commit ID from given file in
```

Comment:
```suggestion
// get tries to fetch the first 7 hex digits of GitHub commit ID from given file in
```
Not strongly set on this, but it reads better for me. Same below.